### PR TITLE
feat(website): add basic landing page

### PR DESCRIPTION
**Pull request by suneettipirneni**
**Please describe the changes this PR makes and why it should be merged:**

Adds a basic landing page, open to design suggestions.

### CODE REVIEW

File: packages/website/src/text.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"heroTitle": "The most popular way to build discord bots.",
```
Comment (Jiralite) [inline packages/website/src/text.json:2]:
```suggestion
	"heroTitle": "The most popular way to build Discord bots.",
```

**Review by Jiralite — COMMENTED** [review]
Query about the code sample - how come `Client` is instantiated as an array of intents?

File: packages/website/src/routes/index.tsx [hunk 2]
```diff
@@ -1,8 +1,47 @@
...
+	);
+}
+
 export default function IndexRoute() {
 	return (
-		<main className="w-full h-screen">
-			<div className="h-screen grid place-content-center">
-				<h1 className="">Coming soon...</h1>
+		<main className="w-full max-w-full max-h-full h-full flex-col">
+			<div className="flex h-[65px] sticky top-0  border-b border-slate-300 justify-center px-10 bg-white">
```
Comment (iCrawl) [inline packages/website/src/routes/index.tsx:20]:
```suggestion
			<div className="flex h-[65px] sticky top-0 border-b border-slate-300 justify-center px-10 bg-white dark:bg-dark">
```

File: packages/website/src/routes/index.tsx [hunk 3]
```diff
@@ -1,8 +1,47 @@
...
+				<div className="flex align-center items-center w-full max-w-[1100px] justify-between">
+					<div className="h-[50px] w-[50px] rounded-lg overflow-hidden ">
```
Comment (iCrawl) [inline packages/website/src/routes/index.tsx:22]:
```suggestion
					<div className="h-[50px] w-[50px] rounded-lg overflow-hidden">
```

File: packages/website/src/routes/index.tsx [hunk 4]
```diff
@@ -1,8 +1,47 @@
...
+						<img className="h-[50px] w-[50px]" src={logo} />
+					</div>
+					<div className="flex flex-row space-x-8">
+						<a className="color-blurple font-semibold">Docs</a>
+						<a className="color-blurple font-semibold">Guide</a>
+					</div>
+				</div>
+			</div>
+			<div className={`flex justify-center w-full max-w-full box-border px-10`}>
```
Comment (iCrawl) [inline packages/website/src/routes/index.tsx:31]:
```suggestion
			<div className="flex justify-center w-full max-w-full box-border px-10">
```

File: packages/website/src/routes/index.tsx [hunk 5]
```diff
@@ -1,8 +1,47 @@
...
+				<div className="flex flex-row grow max-w-[1100px] mt-10 pb-10 space-x-20">
+					<div className="flex flex-col">
+						<h1 className={`font-bold text-6xl max-w-[500px] color-blurple mb-2`}>{text.heroTitle}</h1>
```
Comment (iCrawl) [inline packages/website/src/routes/index.tsx:34]:
```suggestion
						<h1 className="font-bold text-6xl max-w-[500px] color-blurple mb-2">{text.heroTitle}</h1>
```

File: packages/website/src/routes/index.tsx [hunk 6]
```diff
@@ -1,8 +1,47 @@
...
+import logo from '../assets/djs_logo_rainbow_400x400.png';
+import * as text from '../text.json';
+
+interface ButtonProps {
+	title: string;
+}
+
+function Button({ title }: ButtonProps) {
+	return (
+		<div className={`max-h-[70px] bg-blurple px-3 py-4 rounded-lg`}>
```
Comment (iCrawl) [inline packages/website/src/routes/index.tsx:11]:
```suggestion
		<div className="max-h-[70px] bg-blurple px-3 py-4 rounded-lg">
```

File: packages/website/src/routes/index.tsx [hunk 7]
```diff
@@ -1,8 +1,47 @@
...
+			<p className="font-semibold color-white m-0">{title}</p>
```
Comment (iCrawl) [inline packages/website/src/routes/index.tsx:12]:
```suggestion
			<p className="font-semibold text-white m-0">{title}</p>
```

File: packages/website/src/routes/index.tsx [hunk 8]
```diff
@@ -1,8 +1,47 @@
...
-				<h1 className="">Coming soon...</h1>
+		<main className="w-full max-w-full max-h-full h-full flex-col">
+			<div className="flex h-[65px] sticky top-0  border-b border-slate-300 justify-center px-10 bg-white">
+				<div className="flex align-center items-center w-full max-w-[1100px] justify-between">
+					<div className="h-[50px] w-[50px] rounded-lg overflow-hidden ">
+						<img className="h-[50px] w-[50px]" src={logo} />
+					</div>
+					<div className="flex flex-row space-x-8">
+						<a className="color-blurple font-semibold">Docs</a>
+						<a className="color-blurple font-semibold">Guide</a>
```
Comment (iCrawl) [inline packages/website/src/routes/index.tsx:27]:
```suggestion
						<a className="text-blurple font-semibold">Docs</a>
						<a className="text-blurple font-semibold">Guide</a>
```

File: packages/website/src/routes/index.tsx [hunk 9]
```diff
@@ -1,8 +1,47 @@
...
+					</div>
+				</div>
+			</div>
+			<div className={`flex justify-center w-full max-w-full box-border px-10`}>
+				<div className="flex flex-row grow max-w-[1100px] mt-10 pb-10 space-x-20">
+					<div className="flex flex-col">
+						<h1 className={`font-bold text-6xl max-w-[500px] color-blurple mb-2`}>{text.heroTitle}</h1>
+						<p className="max-w-[500px] text-xl color-slate-500">{text.heroDescription}</p>
```
Comment (iCrawl) [inline packages/website/src/routes/index.tsx:35]:
```suggestion
						<p className="max-w-[500px] text-xl text-slate-500">{text.heroDescription}</p>
```

File: packages/website/src/routes/index.tsx [hunk 10]
```diff
@@ -1,8 +1,47 @@
...
+					<div className="h-[50px] w-[50px] rounded-lg overflow-hidden">
+						<img className="h-[50px] w-[50px]" src={logo} />
+					</div>
+					<div className="flex flex-row space-x-8">
+						<a className="text-blurple font-semibold">Docs</a>
+						<a className="text-blurple font-semibold">Guide</a>
+					</div>
+				</div>
+			</div>
+			<div className="flex justify-center w-full max-w-full box-border px-10">
```
Comment (iCrawl) [inline packages/website/src/routes/index.tsx:31]:
```suggestion
			<div className="flex justify-center w-full max-w-full box-border px-10 bg-white dark:bg-dark">
```

**Pull request merged**